add more msvc toolset defines, 14.3 for VS2022, 14.2 for VS2019, 14.1 for VS2017
--- serf-2.0.0/SConstruct
+++ serf-2.0.0/SConstruct
@@ -154,15 +154,20 @@ if sys.platform == 'win32':
     EnumVariable('MSVC_VERSION',
                  "Visual C++ to use for building",
                  None,
-                 allowed_values=('14.2', '14.1', '14.0', '12.0',
-                                 '11.0', '10.0', '9.0', '8.0', '6.0'),
-                 map={'2005' :  '8.0',
+                 allowed_values=('14.3', '14.2', '14.1', '14.0', '12.0',
+                                 '11.0', '10.0', '9.0', '8.0', '7.1', '7.0', '6.0'),
+                 map={
+                      '2002' :  '7.0',
+                      '2003' :  '7.1',
+                      '2005' :  '8.0',
                       '2008' :  '9.0',
                       '2010' : '10.0',
                       '2012' : '11.0',
                       '2013' : '12.0',
                       '2015' : '14.0',
                       '2017' : '14.1',
+                      '2019' : '14.2',
+                      '2022' : '14.3',
                      }),
 
     # We always documented that we handle an install layout, but in fact we
